### PR TITLE
Enhancement: Update localheinz/composer-normalize

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] updates `localheinz/composer-normalize`

💁‍♂️ For reference, see https://github.com/localheinz/composer-normalize/compare/0.6.0...0.9.0.

### CODE REVIEW

**Comment by keradus** [comment]
60 commits without changelog is hard to catch up

please, target 2.12 LTS

File: dev-tools/composer.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
         "php": "^7.1"
     },
     "require-dev": {
-        "localheinz/composer-normalize": "^0.6.0",
+        "localheinz/composer-normalize": "~0.9.0",
```
Comment (keradus) [inline dev-tools/composer.json]: why not caret ?

**Comment by localheinz** [comment]
@keradus 

Good point, should probably start keeping a changelog!

**Comment by localheinz** [comment]
@keradus @SpacePossum 

Since this is admittedly the first time that I have written a changelog, perhaps you can take a look at https://github.com/localheinz/composer-normalize/pull/94 and let me know if this is good? 

I would appreciate it!

**Comment by keradus** [comment]
Thank you @localheinz.

**Pull request merged**